src/PresentationalComponents/ErrState.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import ErrorState from '@redhat-cloud-services/frontend-components/ErrorState';
+
+export const ErrState = ({
+  errorTitle,
+  errorDescription,
+  hrefUrl,
+  hrefText,
+}) => {
+  return (
+    <ErrorState
+      errorTitle={errorTitle}
+      errorDescription={
+        <span>
+          {errorDescription} <br /> <a href={hrefUrl}>{hrefText}</a>
+        </span>
+      }
+    />
+  );
+};
+
+ErrState.propTypes = {
+  errorTitle: propTypes.string,
+  errorDescription: propTypes.string,
+  hrefUrl: propTypes.string,
+  hrefText: propTypes.string,
```

Comment:
:thinking: Maybe we could lift the description composition to the schema and use the `dangerouslySetInnerHTML` React prop. that can open a can of worms but I would assume if we manage the API response then we should be fine on that front.

Then you would use the ErrorState like so:
```jsx

export const ErrState = ({
  errorTitle,
  errorDescription,
}) => {
  return (
    <ErrorState
      errorTitle={errorTitle}
      errorDescription={
        <div dangerouslySetInnerHTML={{__html: errorDescription}}></div>
      }
    />
  );
};

<ErrState errorTitle="Foo" errorDescription="<span>Foo bar <a href="#">baz</a></span>"  />
```

Reply:
Nice suggestion, I pushed up the changes